webapp/components/create_comment.jsx

```diff
@@ -121,15 +119,43 @@ export default class CreateComment extends React.Component {
             return;
         }
 
-        MessageHistoryStore.storeMessageInHistory(this.state.message);
+        MessageHistoryStore.storeMessageInHistory(message);
+
+        if (message.trim().length === 0 && this.state.previews.length === 0) {
+            return;
+        }
+
+        const isReaction = (/^(\+|-):([^:\s]+):\s*$/).exec(message);
```

Comment:
can we have this regex somewhere common to try and avoid not having it update somewhere in case we need to?